### PR TITLE
all sorts of aria labels for bits components

### PR DESCRIPTION
**Pull request by dmytrofirsov**

### CODE REVIEW

File: packages/bits/src/lib/dialog/confirmation-dialog.component.ts [hunk 1]
```diff
@@ -51,4 +57,8 @@ export class ConfirmationDialogComponent {
         return this.setFocus === buttonType ? true : false;
     }
 
+    private getAriaLabel() {
+        return (this.severity ? `${this.severity} ${this.title}` : this.title) || this.ariaLabel
```
Comment (DmytroPutrin) [inline packages/bits/src/lib/dialog/confirmation-dialog.component.ts:61]: add semicolons, plz.

File: packages/bits/src/lib/switch/switch.component.html [hunk 2]
```diff
@@ -4,7 +4,9 @@
              (click)="toggle()"
              (keyup.enter)="toggle()"
              (keyup.space)="toggle()"
-             tabindex="{{disabled ? -1 : 0}}"
+              tabindex="{{disabled ? -1 : 0}}"
+              role="checkbox"
```
Comment (DmytroPutrin) [inline packages/bits/src/lib/switch/switch.component.html:8]: attributes not aligned.

Comment (dmytrofirsov) [reply packages/bits/src/lib/switch/switch.component.html]: done

File: packages/bits/src/lib/dialog/confirmation-dialog.component.ts (hunk 1 above)
Comment (dmytrofirsov) [reply packages/bits/src/lib/dialog/confirmation-dialog.component.ts]: done

**Review by DmytroPutrin — DISMISSED** [review]

File: packages/bits/src/lib/progress/progress.component.ts [hunk 3]
```diff
@@ -40,8 +41,9 @@ export class ProgressComponent implements OnChanges {
     @Output() public cancel = new EventEmitter();
 
     public isIndeterminate = false;
+    public ariaValueNow: string | undefined;
 
-    public ngOnChanges() {
+    public ngOnChanges(changes: SimpleChanges) {
```
Comment (Vita-Karas) [inline packages/bits/src/lib/progress/progress.component.ts:46]: Do we need `changes` here? Look like it is unnecessary.

File: packages/bits/src/lib/selector/selector.component.html [hunk 4]
```diff
@@ -17,6 +19,7 @@
             [displayStyle]="'default'"
             [icon]="'caret-down'"
             (click)="overlay.toggle()"
+            [ariaLabel]="'Open Selector Menu'"
```
Comment (Vita-Karas) [inline packages/bits/src/lib/selector/selector.component.html:22]: Can be changed to `ariaLabel="Open Selector Menu"`

File: packages/bits/src/lib/textbox/textbox.component.ts [hunk 5]
```diff
@@ -36,10 +36,6 @@ import { NuiFormFieldControl } from "../form-field/public-api";
     ],
     styleUrls: ["./textbox.component.less"],
     encapsulation: ViewEncapsulation.None,
-    host: {
-        "role": "textbox",
-        "[attr.aria-label]": "ariaLabel + ' textbox'",
-    },
 })
 export class TextboxComponent implements ControlValueAccessor, NuiFormFieldControl {
     /**
```
Comment (Vita-Karas) [inline packages/bits/src/lib/textbox/textbox.component.ts:41]: Please, remove `@Input() public ariaLabel: string = "";` from textbox. If you don't need ariaLabel anymore.

Comment (dmytrofirsov) [reply packages/bits/src/lib/textbox/textbox.component.ts:41]: aria label is still used in the template

File: packages/bits/src/lib/selector/selector.component.html (hunk 4 above)
Comment (dmytrofirsov) [reply packages/bits/src/lib/selector/selector.component.html:22]: make sense

File: packages/bits/src/lib/progress/progress.component.ts (hunk 3 above)
Comment (dmytrofirsov) [reply packages/bits/src/lib/progress/progress.component.ts]: done

**Review by DmytroPutrin — APPROVED** [review]

**Review by Vita-Karas — DISMISSED** [review]

**Review by DmytroPutrin — APPROVED** [review]

**Review by Vita-Karas — DISMISSED** [review]

**Review by dmytrognatyshyn — DISMISSED** [review]

**Review by Vita-Karas — APPROVED** [review]

**Review by DmytroPutrin — APPROVED** [review]

**Review by dmytrognatyshyn — APPROVED** [review]

**Pull request merged**